cache/in-memory/src/config.rs

```diff
@@ -8,18 +8,31 @@ bitflags! {
     /// created channels, channel updates, and channel deletes, but not their
     /// messages.
     pub struct ResourceType: u64 {
+        /// Information relating to channels.
```

Comment:
It would be nice to list a couple examples of events that make use of each resource type

Reply:
Can I kick this can down the road to when I more specifically check for resource types? It would be easier to list all of them when I'm in that muck